projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -1863,9 +1863,54 @@ public function download_feedback_as_csv() {
 		}
 
 		fclose( $output ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_system_operations_fclose
+
+		$this->record_tracks_event( 'forms_export_responses', array( 'format' => 'csv' ) );
 		exit();
 	}
 
+	/**
+	 * Send an event to Tracks
+	 *
+	 * @param string $event_name - the name of the event.
+	 * @param array  $event_props - event properties to send.
+	 *
+	 * @return null|void
+	 */
+	private function record_tracks_event( $event_name, $event_props ) {
```

Comment:
Can we get rid of that method and use `\Automattic\Jetpack\Tracking()`, like in #29225, since it works on WordPress.com as well:
https://github.com/Automattic/jetpack/pull/29102#issuecomment-1452282596

Reply:
Unfortunately it only works on self hosted sites, not on WordPress.com.